frontend/www/js/omegaup/components/problem/CollectionList.vue

```diff
@@ -9,6 +9,10 @@
         ></omegaup-problem-filter-tags>
         <omegaup-problem-filter-difficulty
           v-model="selectedDifficulty"
```

Comment:
A reserva de lo que comente @lhchavez, creo que ya no será necesario usar el `v-model` porque ya no vamos a esperar a que cambie el valor de los `radio buttons`, cada que elijas un nuevo valor se va a recargar la página con los nuevos parámetros, así que puedes sustituir esta línea:
```suggestion
          :selected-difficulty="difficulty"
```